lib/functions.inc.php

```diff
@@ -317,9 +317,9 @@ function check_password_strength( $password, $oldpassword, $pwd_policy_config, $
     }
 
     # pwned?
-    if ($use_pwnedpasswords) {
+    if ($use_pwnedpasswords and version_compare(PHP_VERSION, '7.2.5') >= 0) {
```

Comment:
We should maybe add a test in htdocs/index.php to return "phpupgraderequired" if `$use_pwnedpasswords and version_compare(PHP_VERSION, '7.2.5') < 0`

Reply:
Will do